chart/instana-agent/README.md

```diff
@@ -285,6 +285,19 @@ The `kubernetes.deployment.pod.requests.cpu`, `kubernetes.deployment.pod.request
 
 ## Changelog
 
+### 1.2.32
+
+* Add support for containerd montoring on TKGI  
+
+### 1.2.31
+
+* Bump leader-elector image to v0.5.14 (Update dependencies)
+
+### 1.2.30
+
+* Pull agent image from IBM Cloud Container Registry (icr.io/instana/agent). No code changes have been made.
```

Comment:
I think this line and the next describe the same thing with little different wording.

Reply:
This copied from the Helm Chart ;-)
Real changes happen only in the following files: 

- VERSION 
- LEADER_ELECTOR_VERSION
- Dockerfile
- chart/instana-agent/templates/application.yaml